src/lib/geo-location.js

```diff
@@ -56,11 +55,14 @@ const getGeoLocation = async ({ mode, offline, state }) => {
     }
   }
 
-  // If the `--geo` option is set to `mock`, we use the mock location. Also,
-  // if the `--offline` option was used, we can't talk to the API, so let's
-  // also use the mock location.
-  if (mode === 'mock' || offline) {
-    return mockLocation
+  // If the `--geo` option is set to `mock`, we use the default mock location.
+  // If the `--offline` option was used, we can't talk to the API, so let's
+  // also use the mock location.  Otherwise, use the country code passed in by
+  // the user.
+  if (!['cache', 'update'].includes(mode) || offline) {
+    return mode === 'mock' && !geoCountry
+      ? mockLocation
+      : { city: '', country: { code: geoCountry, name: '' }, subdivision: { code: '', name: '' } }
```

Comment:
What do you think about populating these fields with some mock data, like "Netlify City" and "Jamstack Avenue"? 😎 

Just so that the strings aren't left empty.

Reply:
Sounds good.  Could also use faker to generate some random data if you want to go that route.  Do you think putting a random two letter subdivision code might be worth considering?

Reply:
I'd avoid faker just because it's an extra dependency for very little gain. Filling in any data should be fine. And yeah, I think I'd try to fill in the entire object. Maybe something like:

```json
{
  "city": "Netlicity",
  "country": <whatever they input>
  "subdivision": {
    "code": "JS",
    "name": "Jamstack"
  }
}
```